src/ui/buttons/buttonDesigns/alternateSlideLogoDesign.jsx

```diff
@@ -44,7 +46,7 @@ export function AlternateSlideLogoComponent({ designLabelText } : ContentOptions
                     opacity: 0; 
                     color: #142C8E;
                     font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-                    font-size: 10px;
+                    padding-top: 1px;
```

Comment:
What's this for?

Reply:
just to center the label text horizontally

Reply:
Does this work with responsive? Is there a better way to center it than padding-top?

Reply:
probably a better way requires more js code, and I am not sure if that is okay